C/plugins/common/include/simple_http.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _SIMPLE_HTTP_H
+#define _SIMPLE_HTTP_H
+/*
+ * FogLAMP HTTP Sender wrapper.
+ *
+ * Copyright (c) 2018 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Massimiliano Pinto, Mark Riddoch
+ */
+
+#include <string>
+#include <vector>
+#include <http_sender.h>
+#include <client_http.hpp>
+
+using HttpClient = SimpleWeb::Client<SimpleWeb::HTTP>;
+
+class SimpleHttp: public HttpSender
+{
+	public:
+		/**
+		 * Constructor: pass host:port & connect & request timeouts
+		 */
+		SimpleHttp(const std::string& host_port,
+				unsigned int connect_timeout = 0,
+				unsigned int request_timeout = 0);
+
+		// Destructor
+		~SimpleHttp();
```

Comment:
Should have a body to deal with m_sender

Reply:
delete m_sender;

is called in simple_http(s).cpp destructor classes